### PR TITLE
#22 fixed

### PR DESCRIPTION
**Pull request by apocarteres**

### CODE REVIEW

**Comment by dmarkov** [comment]
Let me find a reviewer for this pull request, thanks for submitting it

**Comment by dmarkov** [comment]
@krzyk please review this one

File: pom.xml [hunk 1]
```diff
@@ -1669,7 +1669,7 @@
                     to the STDOUT as well, not only to the file.
                     -->
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>2.17</version>
+                    <version>2.18</version>
```
Comment (krzyk) [inline pom.xml]: there is version *2.18.1*, let's use it

**Comment by krzyk** [comment]
@apocarteres just a comment about version

**Comment by apocarteres** [comment]
@krzyk done

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by dmarkov** [comment]
@krzyk I added **15 mins** to your account, many thanks for working with the project!

+15 added to your rating, current score is: [+3634](http://www.netbout.com/b/35035?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**